Gemfile

```diff
@@ -107,6 +107,7 @@ gem "twitter", "~> 7.0" # A Ruby interface to the Twitter API
 gem "uglifier", "~> 4.2" # Uglifier minifies JavaScript files
 gem "validate_url", "~> 1.0" # Library for validating urls in Rails
 gem "vault", "~> 0.16" # Used to store secrets
+gem "warning", "~> 1.2" # Adds custom processing for warnings, including the ability to ignore specific warning messages
```

Comment:
I remember this gem like we just removed it yesterday. _Narrator: he [did remove it yesterday](https://github.com/forem/forem/pull/16665)_.